FinalProject/FinalProject.cpp

```diff
@@ -15,38 +15,43 @@
 using namespace std;
 
 void printMenu(void);
-void reserveFlight(void);
+void reserveFlight(int userId);
 void displayUserReservation(int userId);
 void displayReservation(void);
 void cancalUserFlight(int userId);
 void cancalFlight(void);
 void printFlights(vector<Flight *> flights);
 void printReservation(vector<UserReservation> uReservation);
+bool login(User &user);
 
 Database db;
 
 int _tmain(int argc, _TCHAR* argv[])
 {
+	Database::initDB();

```

Comment:
could you remove initDB() into option like 5, user could choose init it or not, as I dont want  init DB everytime I ran the code, I have to input all data again